### PR TITLE
init test framework

### LINKED ISSUE #521: models 效果和精度持续测试机制及框架
paddle/models 最早定位是作为 Paddle 对外输出模型，便于用户直接修改使用，同时在模型的粒度上验证 paddle 可用性。

目前，models已经积累了相当数量的比较有应用价值的模型，但由于缺少精度和效果的测试机制，难以保证模型的效果，历史上也成出现过版本间效果差异较大的情况。

最新的 fluid 框架完善后，models 需要逐步迁移到 fluid 上，在这之前，我们可以尝试建立 models 的自动测试机制和框架，以实现

- 模型级别的效果验证
- 性能测试
- 宏观上的大功能验证

以长久保证 models 以及 Paddle 的实用性。

### PR DESCRIPTION
**Pull request by Superjomn**
fixes: https://github.com/PaddlePaddle/models/issues/521

### CODE REVIEW

**Review by dzhwinter — APPROVED** [review]
Only change the grammar?

**Comment by dzhwinter** [comment]
Please dismiss the previous Approve, it's a mistake click.

File: consistency/design.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
+# Models continuous integration
```
Comment (kavyasrinet) [inline consistency/design.md:1]: "Models continuous integration" => "Continuous integration for models"

File: consistency/design.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+This framework aims to add continuous integration to formal models based on PaddlePaddle.
+
+## Background
+Currently, there are two official repositories that provide open source models based on PaddlePaddle:
```
Comment (kavyasrinet) [inline consistency/design.md:6]: "based on" => "using"

**Review by kavyasrinet — COMMENTED** [review]
I have a bunch of rephrasing comments, maybe once the other reviewers approve this PR, we can merge this and I can create a new PR with the suggested rephrasing and you folks can review it. I was trying to point them out here, but there are quite a few of them, so I stopped at two.

**Review by wanghaoshuang — COMMENTED** [review]
As a supplement:
We should make standards for feeding arguments to model trainning. 
e.g. ：[save_dir_path](https://github.com/PaddlePaddle/models/blob/develop/generate_chinese_poetry/train.py#L37) of 'generate_chinese_poetry' is different with [model_save_dir](https://github.com/PaddlePaddle/models/blob/develop/text_classification/utils.py#L62) of 'text_classification'.

Furthermore, building models bank as a python module may be helpful for implementing test framework conveniently. After `pip install paddle-models-versionxxx` we can use models as below:
```
import paddle.models
model = models.text_classification()
arguments = {'batch_size': 128, 'train_passes':500 ...}
model.setArgs(arguments)
model.train()
```

**Pull request closed without merging**